spec/spec_helper.rb

```diff
@@ -53,6 +53,9 @@
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  config.filter_run focus: true
+  config.run_all_when_everything_filtered = true
```

Comment:
👍 

Also, another idea. What about we tag all the tests as `courses`, `providers`, etc, as per the endpoint they relate to, so that we change something that changes, say, the `providers` endpoint, we can limit test runs to only those tests that are relevant?

Not a blocker, this would be a nice to have.

Reply:
I'm coming from a place where it took 15-20 mins to run the entire test suite. We had a pretty clear separation of concerns we could tag the tests with, but the problem was that by the time we got there, it was hard to find the motivation to back-fill the tags.

Reply:
I'm happy to not push on this for now, btw.